src/Cache/Errors/Errors.php

```diff
@@ -19,8 +19,21 @@ abstract class MomentoErrorCode
     public const ALREADY_EXISTS_ERROR = "ALREADY_EXISTS_ERROR";
     /**
      * Cache with specified name doesn't exist
+     * @deprecated Use CacheNotFoundError instead
      */
     public const NOT_FOUND_ERROR = "NOT_FOUND_ERROR";
+    /**
+     * Cache with specified name doesn't exist
+     */
+    public const CACHE_NOT_FOUND_ERROR = "NOT_FOUND_ERROR";
+    /**
+     * Store with specified name doesn't exist
+     */
+    public const STORE_NOT_FOUND_ERROR = "STORE_NOT_FOUND_ERROR";
+    /**
+     * Item with specified name doesn't exist
+     */
+    public const ITEM_NOT_FOUND_ERROR = "ITEM_NOT_FOUND_ERROR";
```

Comment:
We should call this `STORE_ITEM...` to disambiguate vs cache items.

Reply:
There is no not found error for cache items-- those are represented as misses. This is also how this is defined in the JS SDK. I'm not necessarily opposed to changing this error code to be specific to storage for future-proofing, but we should be consistent across SDKs. I'm going to leave this for now to maintain consistency with other SDKs but an happy to discuss further.